aten/src/ATen/ScalarType.h

```diff
@@ -11,12 +11,12 @@ namespace at {
 #define AT_FORALL_SCALAR_TYPES(_) \
 _(uint8_t,Byte,i) \
 _(int8_t,Char,i) \
-_(double,Double,d) \
-_(float,Float,d) \
+_(int16_t,Short,i) \
 _(int,Int,i) \
```

Comment:
Not really specific to this PR, but do you know why are we using `int` instead of `int32_t`?

Reply:
I'm not sure why -- @zdevito do you know?